docs/azure/services/Microsoft_Purview_Information_Protection_Overview.md

```diff
@@ -0,0 +1,96 @@
+
+![](../images/PurviewOverview.png)
+
+## Data Discovery
+
+There are three main ways data can be discovered using Purview:
+- Manually - done by users
+- Automated pattern recognition - Purview has a concept of Sensitive information types (SITs) which is used to automatically discover sensitive data based on built-in or custom declared patterns. Microsoft has created approximately 200 built-in patterns to recognise different types of card, healthcare, identity information used across different regions. A list of these patterns and their definitions can be found [here](https://learn.microsoft.com/en-gb/microsoft-365/compliance/sensitive-information-type-entity-definitions?view=o365-worldwide).
+- Machine learning - Pre-trained and custom trainable classifiers are a Purview feature that is used to discover sensitive data based on its content by modelling it based on a sample set of true positive matches to sensitive data to use as training. Microsoft has several pre-trained classifiers and their definitions can be seen [here](https://learn.microsoft.com/en-us/microsoft-365/compliance/classifier-tc-definitions?view=o365-worldwide#trainable-classifiers-definitions). Custom classifiers requires an organisation to prepare a dataset of true positives for a given type of sensitive data and would need some level of training to increase its accuracy before it can be used and published. More information can be found [here](https://learn.microsoft.com/en-gb/microsoft-365/compliance/classifier-learn-about?view=o365-worldwide)
+
+## Data Protection
+
+Once sensitive data is discovered, it needs to be labelled with appropriate sensitivity and retention labels. These labels will configure the classification of that data and therefore prepare it to be protected and governed according to existing compliance and business needs. 
+
+### Sensitivity Labels
+
+New and existing sensitivity labels can be found under the "Information Protection" tab on the compliance portal.
+
+![](../images/InformationProtection.png)
+
+The following actions can be achieved by labelling the data:
+- Encrypt emails, meeting invites, and documents to prevent unauthorized people from accessing this data
+- Mark created content when you use Office apps. This is done by adding watermarks, headers or footers to emails, meeting invites and documents that have a sensitivity label applied.
+- Protect content stored in containers such as sites and groups when you enable the feature to use sensitivity labels with Microsoft Teams, Microsoft 365 groups, and SharePoint sites.
```

Comment:
What do we mean by containers? We should avoid ambiguity even if it might be referred equally as ambiguously in specific parts of the documentation. I think its best we clarify exactly what features in Teams, SharePoint are affected than using containers as a synonym since it might introduce more confusion on what exactly we're talking about